forester/src/operations.rs

```diff
@@ -9,14 +17,26 @@ use solana_client::pubsub_client::PubsubClient;
 use solana_client::rpc_config::RpcAccountInfoConfig;
 use solana_sdk::commitment_config::{CommitmentConfig, CommitmentLevel};
 use solana_sdk::signature::Signer;
+use std::mem;
 use std::sync::Arc;
 use std::time::Duration;
+use tokio::sync::Mutex;
 use tokio::time::sleep;
 
+pub async fn setup_rpc(config: Arc<ForesterConfig>) -> RpcPool<SolanaRpcConnection> {
+    let mut connections: Vec<Arc<Mutex<SolanaRpcConnection>>> = Vec::new();
+    for _ in 0..20 {
```

Comment:
since we want to replace it in the future would it make sense to get this number from config?

Reply:
Added RPC_POOL_SIZE